src/assets/i18n/it/translation.json

```
@@ -0,0 +1,97 @@
+  "processing-consent": "Consenti il processo nel CWA",
+  "bill-status": "stato della fattura",
+  "data-privacy-approve": "Approva il trattamento dei dati",
+  "patientdata-exclude": "Aggiungi i dati del paziente nel QR-Code",
+  "patient-data-correction": "Correggi i dati del paziente",
+  "process-finish": "Finisci il processo",
+  "qr-code": "QR Code",
+  "patient-data": "Dati del paziente",
+  "cancel": "Cancella",
+  "data-submit": "Inserisci i dati",
+  "result-positiv": "Positivo",
+  "result-negative": "Negativo",
+  "result-failed": "Fallito",
+  "process-number": "Pratica numero",
+  "logout": "Disconnetti",
+  "wrong-process-number": "Pratica numero errata",
+  "unknown-process-number": "Pratica numero {{processNo}}, sconosciuta",
+  "server-error": "Errore del sistema: {{status}}",
+  "server-not-reachable": "Il sistema non è raggiungibile",
+  "connection-error": "Errore di connessione: {{message}}",
+  "qr-scan": "Scannerizza il QR-Code",
+  "qr-scan-https-only": "QR-Scan funziona solo con il protocollo https",
+  "qr-code-no-patient-data": "Il codice QR-code non contiene dati del paziente",
+  "error-message": "Messaggio di errore",
+  "serverError": "É accaduto un errore:",
+  "error-patient-data-load": "Non siamo riusciti a caricare i dati del paziente.",
+  "statistics": "Statistiche",
+  "failed-report": "Impossibile caricare la pratica per il test",
+  "totalTestCount": "Test totali",
+  "positiveTestCount": "Totali Test Positivi",
+  "successfull-transferred": "Dati caricati correttamente",
+  "record-vaccination-cert-dat": "Certificazione della Pratica di Vaccinazione",
+  "vaccination-cert": "Certificato di Vaccinazione",
+  "identifierType": "Tipo di Identificazione",
+  "country": "Nazionalitá del paziente",
+  "identifierNumber": "Numero identificativo",
+  "disease-agent": "Malattia / Agente",
+  "def-disease-agent": "Malattia o Agente affetto (e.g. 840539006)",
+  "vaccine": "Vaccino/Prophylaxis",
+  "def-vaccine": "Descrizione generica del vaccino/prophylaxis o dei suoi componenti (e.g. 1119305005)",
+  "vac-medical-product": "Prodotto medico",
+  "def-vac-medical-product": "Codice del medicinale (e.g. EU/1/20/1528)",
+  "vac-marketing-holder": "Sistema di gestione delle organizzazioni",
+  "def-vac-marketing-holder": "Codice definito nell'EMA SPOR - Sistema di gestione delle organizzazioni (e.g. ORG-100030215)",
+  "sequence": "Numero di dose",
+  "def-sequence": "Numero di dosi administrate in un ciclo (e.g. 2)",
+  "tot": "Serie totale di dosi",
+  "def-tot": "Numero di dosi previste per il completamento del ciclo (e.g. 1)",
+  "vac-last-date": "Data di vaccinazione",
+  "vac-country": "Stato emittente",
+  "def-vac-country": "Stato Membro",
+  "lot": "Indentificazione",
+  "def-lot": "Una combinazione distintiva di numeri e / o lettere che identifica specificamente un lotto",
+  "certificateIssuer": "Emittente del Certificato",
+  "def-adm": "Nome o Codice dell'administratore del centro (e.g. Regione Lazio)",
```

```suggestion
  "def-adm": "Nome o Codice del Centro di Somministrazione (e.g. Regione Lazio)",
```
Note that one meaning of administer is "to give a medication" (see https://en.wiktionary.org/wiki/administer )